app/src/main/java/com/zulip/android/activities/LoginActivity.java

```diff
@@ -233,15 +233,17 @@ public void onClick(View v) {
             case R.id.google_sign_in_button:
                 connectionProgressDialog.show();
                 saveServerURL();
+                Toast.makeText(this, getString(R.string.logging_into_server, ZulipApp.get().getServerURI()), Toast.LENGTH_SHORT).show();
```

Comment:
this message is for sign ins that are to servers other than Zulip.  here, it'll only show for G+ logins, which is incorrect.